ui-v2/src/components/concurrency/task-run-concurrency-limit-details/task-run-concurrency-limit-details.tsx

```diff
@@ -0,0 +1,67 @@
+import { cn } from "@/lib/utils";
+
+import type { TaskRunConcurrencyLimit } from "@/hooks/task-run-concurrency-limits";
+import { format, parseISO } from "date-fns";
+import React from "react";
+
+type Props = {
+	data: TaskRunConcurrencyLimit;
+};
+
+export const TaskRunConcurrencyLimitDetails = ({ data }: Props) => {
+	return (
+		<div className="flex flex-col gap-4">
+			<dt className="text-muted-foreground">Tag</dt>
+			<dd className="text-sm font-medium leading-none">{data.tag}</dd>
+			<hr />
+			<dl className="flex flex-col gap-2">
+				{getKeyValueList(data).map((d) => (
+					<React.Fragment key={d.label}>
+						<dt className="text-muted-foreground">{d.label}</dt>
+						<dd
+							className={cn(
+								"text-sm font-medium leading-none",
+								d.formatType === "date" && "font-mono",
+							)}
+						>
+							{d.value}
+						</dd>
+					</React.Fragment>
+				))}
+			</dl>
+		</div>
```

Comment:
This should probably be wrapped in a `Card`.

Reply:
This component is just for the side details.

The card component is in [this PR](https://github.com/PrefectHQ/prefect/pull/16444)

<img width="1249" alt="Screenshot 2024-12-24 at 8 47 21 PM" src="https://github.com/user-attachments/assets/eef061d3-058c-4990-8b5c-945ee50bb302" />

Reply:
Yeah, I guess I thought it'd look better wrapped in a card, but the current version doesn't use a card here, so nevermind!